awesomemotive/easy-digital-downloads
# Remove dev assets from the build #9277

Fixes #9277 

Proposed Changes:
1. Adjusts the Grunt build process to only copy over the built assets, not the development assets.

As a note, this brought the final build .zip from 3.6MB to 3.2MB, which is a pretty good amount of savings.

To test:
1. Checkout `issue/9277`
2. Run `npm install`
3. Run `npm run biuld`
4. Copy the `build/easy-digital-downloads` folder to a clean WordPress install and activate EDD.
5. Ensure that Styles and JavaScript assets run, you can checkout, tax settings work, edit/add new orders work.
